### DISCUSSION TITLE
Keep Your Sandbox Tight! – Actually just a collection of my own tight sandbox configurations.

### DISCUSSION THREAD

**Discussion by rusty-snake** [netblue30/firejail#4099] [Show and tell]
Or simple: I've finally managed to upload my firejail locals and flatpak overrides.

Have a look at [kyst](https://github.com/rusty-snake/kyst/) if you want to take some of them.

**Comment by reinerh** [comment]
Thanks!

I have a few AppArmor profiles for proprietary applications I sometimes use, and Firefox (which requires more security as it's my main browser): https://codeberg.org/reinerh/apparmor-profiles

**Comment by netblue30** [comment]
I would say just merge them in - I mean all of them!

If something breaks we'll deal with them as people start complaining.

**Reply by rusty-snake** [reply]
There are only a few (mostly D-Bus policies) which can be upstreamed. The vast majority of the tightens will break features (that I don't need/use).
